package.json

```diff
@@ -29,7 +29,9 @@
     "api-documenter": "api-documenter yaml --input-folder=temp"
   },
   "dependencies": {
-    "google-gax": "^2.1.0"
+    "fs.promises": "^0.1.2",
+    "google-gax": "^2.1.0",
+    "p-queue": "^6.6.1"
```

Comment:
We want `p-queue` in `samples/package.json`, but not here (apologies).  You need to `cd` into `samples` and run `npm install p-queue`

Reply:
Done.